src/components/MoviePage/MoviePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import HeaderContainer from '../HeaderContainer/HeaderContainer';
+import SearchResultsContainer from '../SearchResultsContainer/SearchResultsContainer';
+
+export default class MoviePage extends React.PureComponent {
```

Comment:
General Note! Need to double-check usage of Class and Functional components.

Reply:
I looked through the project, and now it seems that I have changed components where it was necessary (from my point of view)